### DISCUSSION TITLE
SharpDX.SharpDXException: "HRESULT: [0x887A0005], Module: [SharpDX.DXGI], ApiCode: [DXGI_ERROR_DEVICE_REMOVED/DeviceRemoved], Message: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.

### DISCUSSION THREAD

**Discussion by zver322** [stride3d/stride#2057] [Q&A]
Hello, I'm using Stride Shaders in my project and I faced an error. 
I'm passing a buffer of Vector3 and its size is 2097 x 2097 (total elements size is 4,397,409 with elementSize 12). This calculates without errors.
When I'm passing a buffer of Vector3 and its size is 4097 x 4097 (total elements size is 16,785,409 with elementSize 12). After a quite long calculation time (~2 mins) I'm trying to get data from buffer and set to an array I got and error which is typed in title. My GPU is NVidia 3080 and  BuffersMemory = 1274445216.
Can it be a problem with ComputeEffectShader.Draw or can it be GPU buffer overflow?

![image](https://github.com/stride3d/stride/assets/84985250/945ee9b3-530b-44a2-98eb-665ed3fde7e8)
![image](https://github.com/stride3d/stride/assets/84985250/3067c2c7-e7b3-4ade-9e86-bb84cc45b3cb)

**Comment by AnswerHJW** [comment]
same with me
